packages/editor/src/context/DndContext.tsx

```diff
@@ -70,7 +70,11 @@ export const DndContext = ({ componentByName, children }: { componentByName: Com
   };
 
   const mouseSensor = useSensor(MouseSensor, { activationConstraint: { distance: 15 } });
-  const sensors = useSensors(ui.helpPaddings ? mouseSensor : undefined);
+  const sensors = useSensors(mouseSensor);
+
+  if (!ui.helpPaddings) {
+    return <>{children}</>;
+  }
```

Comment:
fix react warning about changing order in somewhere useEffect.
Just disable the dnd in preview mode by not render the DndContext